index.html

```diff
@@ -204,7 +205,38 @@ <h3>Example</h3>
     "lprNumber": "999-999-999",
     "commuterClassification": "C1",
     "birthCountry": "Bahamas",
-    "birthDate": "1958-07-17"
+    "birthDate": "1958-07-17",
+    "height": [
+      {
+        "type": ["QuantitativeValue"],
+        "unitCode": "ft",
+        "value": 5
```

Comment:
This would certainly be simpler with just a single amount in the smallest unit, i.e., just an inches value. Software could do whatever calculations are necessary to express inches in feet if desirable.